src/main/java/orangebeans/BOGOPromotionService.java

```diff
@@ -0,0 +1,21 @@
+package orangebeans;
+
+import orangebeans.models.Order;
+import orangebeans.models.Product;
+import orangebeans.models.Promotion;
+
+public class BOGOPromotionService implements PromotionService {
+
+	public void applyPromotion(Order order, Product product) {
+		final Promotion promotion = product.getPromotion();
+		final Integer offer = promotion.getOffer();
```

Comment:
why does `promotion.getOffer()` returns integer. Need to find better way to associate offer